### PR TITLE
[GITHUB-6174] Support external-dns 0.5.4

### PR DESCRIPTION
**Pull request by quater**
* Amended RBAC to allow listing nodes
* Bumped Docker image version to 0.5.4

### CODE REVIEW

File: stable/external-dns/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 0.6.1
-appVersion: 0.5.0
+version: 0.6.2
+appVersion: 0.5.3
```
Comment (linki) [inline stable/external-dns/Chart.yaml]: `0.5.3` has a bug 😞  let's wait for `0.5.4` or go with `0.5.2`.

Comment (coreypobrien) [reply stable/external-dns/Chart.yaml]: Agreed. We don't need this node list permission until 0.5.3, but we should wait for 0.5.4 to get the fixes for the cache bugs that got introduced in 0.5.3.

**Review by coreypobrien — CHANGES_REQUESTED** [review]

File: stable/external-dns/Chart.yaml (hunk 1 above)
Comment (starkers) [reply stable/external-dns/Chart.yaml]: :+1:  just tried 0.5.3 and hit bugs

Comment (njuettner) [reply stable/external-dns/Chart.yaml]: We released 0.5.4

Comment (njuettner) [reply stable/external-dns/Chart.yaml]: https://github.com/kubernetes-incubator/external-dns/releases/tag/v0.5.4

**Comment by andreasolund** [comment]
Ping @quater, could you update your PR according to @njuettner comments?

**Comment by quater** [comment]
@linki @coreypobrien @andreasolund I have bumped the version to 0.5.4 now.

**Comment by quater** [comment]
Apologies, this PR was automatically closed when I deleted the remote branch in my GitHub account. Here is the replacement PR https://github.com/helm/charts/pull/6855.

**Pull request closed without merging**